src/stores/appcatalog/selectors.ts

```diff
@@ -13,9 +14,40 @@ export function selectCatalogs(state: IState): IAppCatalogsState {
   return state.entities.catalogs;
 }
 
+export function selectSelectedCatalogs(state: IState): string[] {
+  return Object.entries(state.entities.catalogs.ui.selectedCatalogs)
+    .filter(([_, enabled]) => enabled)
+    .map(([catalogName]) => catalogName);
```

Comment:
Not a big deal, but we can do this in a single loop using `.reduce()` or just a regular `for` loop